src/test/java/com/epam/geometry/DirectorTest.java

```diff
@@ -1,5 +1,42 @@
 package com.epam.geometry;
 
+import com.epam.geometry.data.DataException;
+import com.epam.geometry.data.DataReader;
+import com.epam.geometry.model.Point;
+import com.epam.geometry.model.Triangle;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 // Write tests using Mockito to mock dependencies
 public class DirectorTest {
+
+    @Test
+    public void testProcessShouldRunNormallyWhenValidInput() throws DataException {
+
+        Point a = new Point(1.0, 2.0);
+        Point b = new Point(3.0, 4.0);
+        Point c = new Point(5.0, 6.0);
+
+        List<Triangle> expected = Arrays.asList(new Triangle(a, b, c));
```

Comment:
for test you can write Arrays.asList(new Triangle(              
                                                                 new Point(1.0, 2.0),
                                                                 new Point(1.0, 2.0), 
                                                                 new Point(1.0, 2.0)));